### PR TITLE
Maxent pipeline: some docs, bug fix, defaults and simplified create background function

### PR DESCRIPTION
**Pull request by frousseu**
Described a bit more some arguments and functions, simplified the create background function and added the thickening and inclusion_buffer method which were not working/available, now accepts the - in soilgrids variable names, reduced the resolution for faster runs

### CODE REVIEW

File: pipelines/SDM/SDM_maxEnt.json [hunk 1]
```diff
@@ -896,5 +899,17 @@
         1
       ]
     }
+  },
+  "metadata": {
+    "name": "Maxent SDM pipeline",
+    "description": "A pipeline to produce species distribution models using Maxent",
+    "author": [
+      {
+        "name": "Sarah Valentin",
```
Comment (jmlord) [inline pipelines/SDM/SDM_maxEnt.json:908]: François: shouldn't you be in the authors too now?

**Review by jmlord — COMMENTED** [review]
Someone needs to review the R code

**Comment by tpoisot** [comment]
Any thoughts about using [maxnet](https://github.com/mrmaxent/maxnet) instead of `maxent.jar`? The differences are covered in the paper (https://onlinelibrary.wiley.com/doi/10.1111/ecog.03049), and the results are about the same, with no need for an extra java (closed-source) dependency?

**Comment by frousseu** [comment]
From my limited experience, maxent.jar seems faster and less buggy than maxnet, but I agree maxnet is more open-source and has fewer dpendencies. I changed the algorithm to maxent.jar, because it scales variables behing the scene, while maxnet does not and I was running into numerical problems resulting from non-scaled variables. We could easily scale variables ourselves, but it can be a bit of a pain when exploring results and I wanted to avoid that. Another option could be to wait for maxnet to scale variables https://github.com/mrmaxent/maxnet/issues/24 (or to submit a pull request to maxnet to allow for scaling in glmnet).

**Review by glaroc — APPROVED** [review]

**Pull request merged**